stdlib/char.gr

```diff
@@ -315,64 +315,170 @@ provide let (>=) = (x: Char, y: Char) => {
 }
 
 /**
- * Checks if the character is an ASCII digit.
+ * Sub module for working with ASCII characters.
  *
- * @param char: The character to check
- * @returns `true` if the character is an ASCII digit or `false` otherwise
- * 
- * @example assert Char.isAsciiDigit('1')
- * @example assert !Char.isAsciiDigit('a')
+ * @example Char.Ascii.isAscii('1')
```

Comment:
I think it's fine to do the examples as just `Ascii.` without the leading `Char.`.

Reply:
I think we should keep the `Char.Ascii.isAscii`, I feel like it's not very long and it makes the examples runnable, in other modules such as array I was using the pattern:
```
use Array.{ module Immutable }
assert Immutable.isEmpty(Immutable.empty) == true
```
but I felt that the module name ascii was small enough here to just directly inline it.